docs/RationaleSeq.md

```diff
@@ -99,6 +99,93 @@ optimization) another op could be added.
   targets "reset-agnostic code", the reset style shouldn't affect logical
   correctness. It should, therefore, be determined by a lowering pass.
 
+## The FIRRTL register operation [Provisional]
+
+The `seq.firreg` carries all the information required to represent a FIRRTL
+register and lower it to SystemVerilog.
+
+`FirReg` has the following operands:
+
+- **input**: Value to set the register to on the positive edge of the clock
+signal.
+- **clk**: Clock signal driving the register.
+- **name**: A name for the register, passed directly to the `sv.reg`
+- **inner_sym**: A symbol for the register, passed directly to the `sv.reg`
+- **reset**: Signal to trigger the reset.
+- **resetValue**: A value which is set upon reset. Must be a constant if the
+reset is asynchronous.
+- **isAsync**: Optional boolean flag indicating whether the reset is
+asynchronous.
+- **randomized**: Boolean flag indicating whether the register it initialized
+with a random value.
+
+All registers are given a random preset value.
```

Comment:
Is it all registers, or is the `randomized` flag used to control this? This statement seems to conflict with having a flag.

Reply:
Thanks for spotting this! The comment was outdate.